packages/react-dom/src/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -121,105 +143,106 @@ describe('ReactCompositeComponent-state', () => {
     };
   });
 
-  it('should support setting state', () => {
+  it('should support setting state', async () => {
```

Comment:
This test is so old, it's mostly unchanged since the initial release. Pretty cool that `createRoot` 10 years later doesn't alter the order of the logs here.